src/smooch.js

```diff
@@ -16,15 +16,22 @@ import { decode } from 'jsonwebtoken';
 
 import packageInfo from '../package.json';
 
-const SERVICE_URL = 'https://api.smooch.io/v1';
+const SERVICE_URL = 'https://api.smooch.io';
 
 if (!global.FormData) {
     global.FormData = require('form-data');
 }
 
 class Smooch {
-    constructor(auth = {}, options = {}) {
-        const {serviceUrl = SERVICE_URL, headers = {}, httpAgent} = options;
+    constructor(options = {}) {
```

Comment:
What was the issue with separating auth from options?

Reply:
I think there just wasn't any good reason for them being split in the first place. `httpAgent ` was the only documented 2nd param option, so I'm taking the opportunity to improve the design a bit before we get stuck with having `serviceUrl` in a 2nd params object.

I suppose it wouldn't have been an issue if javascript had named parameters.